docker-for-mac/index.md

```diff
@@ -215,84 +200,76 @@ no_proxy=*.local, 169.254/16
 
 You can see from the above output that the `HTTP_PROXY`, `http_proxy`, and
 `no_proxy` environment variables are set. When your proxy configuration changes,
-Docker restarts automatically to pick up the new settings. If you have
-containers that you wish to keep running across restarts, you should consider
-using [restart policies](/engine/reference/run/#restart-policies-restart).
+Docker restarts automatically to pick up the new settings. If you have any
+containers that you would like to keep running across restarts, you should consider using [restart policies](/engine/reference/run/#restart-policies-restart).
 
-<p id="daemon-experimental-mode" />
+#### Network
 
-### Daemon
+You can configure Docker Desktop networking to work on a virtual private network (VPN). Specify a network address translation (NAT) prefix and subnet mask to enable Internet connectivity.
 
-You can configure options on the Docker daemon that determine how your
-containers run.
+![Network settings](images/menu/prefs-network.png)
 
-Select **Basic** to configure the daemon with interactive settings, or select
-**Advanced** to edit the JSON directly.
+### Docker Engine
 
-![Daemon](images/menu/prefs-daemon-basic.png){:width="400px"}
+The Docker Engine page allows you to configure the Docker daemon to determine how your containers run.
 
+![Daemon](images/menu/engine.png)
 
-#### Experimental features
+Type a JSON configuration file in the box to configure the daemon settings. For a full list of options, see the Docker Engine [dockerd commandline
+reference](/engine/reference/commandline/dockerd.md){:target="_blank"
+class="_"}.
 
-Both Docker Desktop for Mac Stable and Edge releases have experimental features enabled
-on Docker Engine, as described [Docker Experimental Features
-README](https://github.com/docker/docker-ce/blob/master/components/cli/experimental/README.md){:
-target="_blank" class="_"}. If you uncheck **experimental mode**, Docker Desktop for Mac
-uses the current generally available release of Docker Engine.
+Click **Apply & Restart** to save your settings and restart Docker Desktop.
 
-> Don't enable experimental features in production
->
->Experimental features are not appropriate for production environments or
->workloads. They are meant to be sandbox experiments for new ideas. Some
->experimental features may become incorporated into upcoming stable releases,
->but others may be modified or pulled from subsequent Edge releases, and never
->released on Stable.
+### Command Line
+
+On the Command Line page, you can specify whether or not to enable experimental features.
+
+![Command Line](images/menu/command-line.png)
+
+{% include experimental.md %}
+
+On both Docker Desktop Edge and Stable releases, you can toggle the experimental features on and off. If you toggle it off, Docker Desktop uses the current generally available release of Docker Engine.
```

Comment:
Suggest changing the second sentence to "If you toggle the experimental features off, ..."